cnd/src/http_api.rs

```diff
@@ -526,11 +517,11 @@ mod tests {
             price: Amount::dai(Erc20Quantity::from_wei_dec_str("9100000000000000000000").unwrap()),
             quantity: Amount::btc(Bitcoin::from_sat(10000000)),
             state: State {
-                open: 30,
-                closed: 10,
-                settling: 0,
-                failed: 60,
-                cancelled: 0,
+                open: Amount::btc(Bitcoin::from_sat(3000000)),
```

Comment:
```suggestion
                open: Amount::btc::from_sat(3000000),
```
Surely, if such function does not exists, we need to create it.